Packs/PAN-OS/Integrations/Panorama/Panorama_test.py

```diff
@@ -1509,6 +1576,52 @@ def test_download_software(self, mock_topology):
             for value in result_dataclass.__dict__.values():
                 assert value
 
+    def test_reboot(self, mock_topology):
+        """
+        Test the reboot function returns the corect data
+        The pan-os-python reboot method actually doesn't return any output itself unless it errors, so we just check our
+        dataclass is set correctly within the function and returned by this function.
+        """
+        from Panorama import UniversalCommand
+
+        result = UniversalCommand.reboot(mock_topology, MOCK_PANORAMA_SERIAL)
+        # Check all attributes of summary data have values
+        for result_dataclass in result.summary_data:
+            for value in result_dataclass.__dict__.values():
+                assert value
+
+        # We also want to check that if an empty string is passed, an error is returned
+        with pytest.raises(
+                DemistoException,
+                match="filter_str  is not the exact ID of a host in this topology; use a more specific filter string."
```

Comment:
the space here is in purpose?

Reply:
Yes, this is the error message you get if you somehow pass an empty string into the filter string.